Workspace/src/arm_interface/src/arm_motor_commands_node.cpp

```diff
@@ -0,0 +1,110 @@
+#include <ros/ros.h>
+#include <std_msgs/String.h> 
+#include <std_msgs/Int32.h>
+#include <stdbool.h>
+#include <list>
+
+struct msg_from_topics {
+	bool ik_status; 
+	std::list<float> data_points; 
+};
+
+class InitObjects {
+	private:
+	ros::init(argc, argv, "arm_motor_commands");
```

Comment:
You can’t call functions in a class definition like this. Call these lines from a constructor

Reply:
Or an initializer function